src/Audit/AuditIQServer.ts

```diff
@@ -16,32 +16,19 @@
 import { IqServerResult } from "../Types/IqServerResult";
 import { ReportStatus } from "../Types/ReportStatus";
 import chalk = require("chalk");
+import { visuallySeperateText } from "../Visual/VisualHelper";
 
 export class AuditIQServer {
   public auditThirdPartyResults(results: ReportStatus): boolean {
     if (results.isError) {
-      console.log();
-      console.group();
-      console.error(results.errorMessage);
-      console.groupEnd();
-      console.log();
+      visuallySeperateText(results.errorMessage);
```

Comment:
just checking this will just then be a console.log instead of the console.error (as before)?

Reply:
It comes down to if we want stuff going to `stderr` or `stdout`. Extended to do so in: 9c7bdf4